app/views/api_docs/pages/home.md

```diff
@@ -135,27 +134,30 @@ The number after the decimal point indicates a minor version. This is
 incremented each time non-breaking changes are made, for example `1.2` changes to
 `1.3`.
 
-The current version of this API is `0.1`. The next minor version will be `0.2`.
-
-Changes are documented in our release notes.
+Changes are documented in our [release notes](/api-docs/release-notes).
 
 ## Using the correct version of the API
 
 When an API version is officially released, minor version updates will be made
 available:
 
-- on their own minor version URL, for example `v1.1` on a major version URL which
-- does not indicate a minor version, for example `v1` This means that if you use
-  the major version URL, you do not need to update student record systems every
-  time we make a minor update.
+- on their own minor version URL, for example `v1.1`
+- on a major version URL which does not indicate a minor version, for example `v1`
+
+This means that if you use the major version URL, you do not need to update student
+record systems every time we make a minor update.
 
 For example, after version 1.1 is released you can use:
 
 - <https://www.register-trainee-teachers.service.gov.uk/api/v1.0> for version `1.0`
 - <https://www.register-trainee-teachers.service.gov.uk/api/v1.1> for version `1.1`
-- <https://www.register-trainee-teachers.service.gov.uk/api/v1> for version `1.1` - but if version `1.2` is released then this URL will give you version `1.2` instead
+- <https://www.register-trainee-teachers.service.gov.uk/api/v1> for version `1.1` - but
+if version `1.2` is released then this URL will give you version `1.2` instead
 
 # Testing
 
-To get familiar with our system and perform testing, you can use our
-[sandbox environment](https://sandbox.register-trainee-teachers.service.gov.uk/).
+To get familiar with our system and perform testing, you can use
+[our sandbox environment](https://sandbox.register-trainee-teachers.service.gov.uk/).
+
+You will need an [authentication token](http://localhost:3000/api-docs/reference#authentication)
```

Comment:
Should it use localhost here?

Reply:
No, it shouldn't. Fixed in https://github.com/DFE-Digital/register-trainee-teachers/pull/4133/commits/f2fb052d35e4f7c2da03676ba4b39a7fa9b3c3d7